roles/open_zaak_podman/tasks/setup_env.yml

```diff
@@ -29,32 +35,10 @@
 # the podman user. However, nginx on the host system serves the files for private media
 # uploads, which runs as the nginx user.
 # Therefore, we change the group of /home/podman/.local/share/containers/storage/volumes
-# to add the +x permission, and add nginx to that group.
-
-- name: Create group to enter container volumes directory
-  group:
-    name: enter-container-volumes
-    state: present
+# to add the +x permission, and add nginx to the podman group.
 
-- name: Ensure nginx user can enter container volumes
+- name: Ensure nginx user is member of podman group
```

Comment:
Doesn't this imply that nginx is able to manage containers? That would be a concern for the violation of least-principle :thinking:

Reply:
It does not (apparently). I have tried, but unable to accomplish anything as the nginx user (no pods show, no systemd, etc). I would actually be more concerned we are creating a "real" nginx user with a shell and potential ssh access. 

Nginx user creation should be handled by nginx installation imho.